build.xml

```diff
@@ -325,4 +325,32 @@
         </exec>
     </target>
 
+    <target name="check-correct-yaml-indents" description="Run yaml standards checks">
+        <exec
+                executable="${path.yaml-standards}"
+                logoutput="true"
+                passthru="true"
+                checkreturn="true"
+        >
+            <arg value="./.github/" />
```

Comment:
please change it into
```
.  \
--exclude-dir=./microservices/product-search-export/vendor \
--exclude-dir=./microservices/product-search/vendor \
--exclude-dir=./vendor \
--exclude-dir=./project-base/var/postgres-data \
--exclude-dir=./project-base/node_modules
```

Reply:
maybe some more description that not the same way but each line into `<arg value="." />` xml style

Reply:
try to create the same yaml-standards target in project-base/build-dev.xml as it is in ./build.xml and that in ./build.xml it will rewrite with project-base/ relative path things